package.json

```diff
@@ -78,8 +78,10 @@
     "url": "git+https://github.com/mozilla/blurts-server.git"
   },
   "scripts": {
-    "start": "node is-v2.js && npm start -w src || npm run build && node server.js",
-    "dev": "node is-v2.js && (cd src && nodemon app.js || exit 0) || nodemon server.js",
+    "start": "npm start -w src",
```

Comment:
@toufali Are you working with @bwells-moz to update the prod deploy to use `start:legacy` instead?

Reply:
You're fast!  I'm writing up a slack comment to open discussion on this right now 😉